src/features/island/buildings/components/building/WithCraftingMachine.tsx

```diff
@@ -15,12 +15,15 @@ const isIdle = (state: MachineState) => state.matches("idle");
 const isCrafting = (state: MachineState) => state.matches("crafting");
 const isReady = (state: MachineState) => state.matches("ready");
 const itemName = (state: MachineState) => state.context.name;
+const secondsTillReady = (state: MachineState) =>
```

Comment:
We will need to remove this as well as the `const secondsLeft = useSelector(craftingService, secondsTillReady);` below as this is what subscribes us to this value.

Reply:
Removed